src/msg/msg_types.h

```diff
@@ -197,14 +197,17 @@ static inline void decode(sockaddr_storage& a,
  * an entity's network address.
  * includes a random value that prevents it from being reused.
  * thus identifies a particular process instance.
- * ipv4 for now.
+ *
+ * This also happens to work to support cidr ranges, in which
+ * case the nonce contains the netmask. It's great!
  */
 struct entity_addr_t {
   typedef enum {
     TYPE_NONE = 0,
     TYPE_LEGACY = 1,  ///< legacy msgr1 protocol (ceph jewel and older)
     TYPE_MSGR2 = 2,   ///< msgr2 protocol (new in ceph kraken)
     TYPE_ANY = 3,  ///< ambiguous
+    TYPE_CIDR = 4,
```

Comment:
Should update `operator<<` to print CIDR `entity_addr_t`  appropriately. But how should it look? A nonce already looks like a CIDR network.

Reply:
operator<< includes the type, so it'll display as "cidr: 10.2.1.100/24" or whatever. That seems good to me?